python-clusters/create-gke-cluster/cluster.json

```diff
@@ -77,6 +77,13 @@
             "mandatory": false,
             "visibilityCondition": "model.isVpcNative"
         },
+        {
+            "name": "nodepoolTags",
+            "type": "STRINGS",
+            "label": "Network tags",
+            "description": "Network tags to each node",
```

Comment:
Maybe re-use GCP's description :
`Tags represent firewall rules applied to each node.`